server/src/main/java/org/apache/iotdb/db/metadata/mnode/MNode.java

```diff
@@ -127,4 +201,43 @@ public String getName() {
   public void setName(String name) {
     this.name = name;
   }
+
+  public void setChildren(Map<String, MNode> children) {
+    this.children = children;
+  }
+
+  public void serializeTo(BufferedWriter bw) throws IOException {
+    serializeChildren(bw);
+
+    String s = String.valueOf(MetadataConstant.MNODE_TYPE);
+    s += "," + name + ",";
```

Comment:
ah..... String.format or StringBuilder?

Reply:
Actually it's the IDE that recommends me to use `String` instead of `StringBuilder`...
![image](https://user-images.githubusercontent.com/19167280/84757951-6258ef80-aff7-11ea-9d29-2a4e1be43f83.png)

Reply:
I'll try `Striing.format`

Reply:
And another serialization method has the same issue.
My another question is,... seems these serialize() methods are not used...

Reply:
Fixed. These methods will be used in my next PR for [issue IOTDB-726](https://issues.apache.org/jira/browse/IOTDB-726)

Reply:
Emm...  As is mentioned in [this page](https://stackoverflow.com/questions/44117788/performance-between-string-format-and-stringbuilder) which compares the performance between String.format and StringBuilder thanks to @Alima777 , I'll use StringBuilder anyway.